docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+services:
+  eventsapi:
+    build:
+      context: .
+    volumes:
+      - .:/code
+    ports:
+      - "9000:80"
```

Comment:
@mjoss17 , @P-Gill97 - Any thoughts on choice of this port? I skipped to `9xxx` since we already use so many in the `8xxx` space for other services typically running in our dev environments, and I didn't want to think too hard about a value that wouldn't collide :joy: . It's pretty arbitrary so I'm open to alternate suggestions.

Reply:
Actually, as one alternative I can reclaim `8888` by killing off the `eventsapi` in the [spikes docker environment](https://github.com/openstax/raise-moodle-spikes/blob/b8621988a29c580e0c39bf8270ef775455e34c67/docker-compose.yml#L47-L57). That might be useful to avoid confusion anyways :man_shrugging: .

Reply:
@rnathuji I'm ambivalent about the port number, but I agree that it would be nice/clearer to have the eventsapi killed off just to avoid confusion. If that's easily done now - I'm cool with it 👍

Reply:
:+1:  - I think I'll go with using `8888` and removing the service from the spikes configuration. I'll update this PR + create one there accordingly.